src/30_Advanced/SeatMap.html

```diff
@@ -0,0 +1,272 @@
+      margin: 8px;
+      width: 40px;
+      height: 40px;
+      padding: 0;
+      border-width: 0;
+      box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.25);
+      background-color: white;
+      border-radius: 3px;
+      }
+      .opaque {
+      opacity: 0.5;
+      filter: alpha(opacity=50);
+      }
+      #choose-seats-button {
+      padding-bottom: 15px;
+      padding-right: 10px;
+      }
+      .seat-map-svg {
+      padding: 0 30px;
+      box-sizing: border-box;
+      }
+      .hide {
+      display: none;
+      }
+      .list-container {
+        position: relative;
+        height: 70vh;
+      }
+      .ampstart-card {
+        min-width: 100%;
+      }
+    </style>
+    <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style>
+    <noscript>
+      <style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style>
+    </noscript>
+
+  </head>
+  <body>
+    <!-- ## How to implement a seat map -->
+    <!-- Here we are using an `amp-state` called `user`
+    to keep a list of selected seat ids in the `selectedSeats` array -->
+    <amp-state id="user">
+      <script type="application/json">
+        {
+          "selectedSeats": []
+        }
+      </script>
+    </amp-state>
+    <!-- `transform` state is used to keep track of the scale of the content being zoomed using by `amp-pan-zoom`. -->
+    <amp-state id="transform">
+      <script type="application/json">
+        {
+          "scale": 1
+        }
+      </script>
+    </amp-state>
+
+    <!-- `zoom` state is used to keep track of the seat map is in a zoomed state or not. This variable is used to
+    decide whether to show a + or - button -->
+    <amp-state id="zoom ">
+          <script type="application/json">
+            0
+          </script>
+    </amp-state>
+
+    <h1>Seat map</h1>
+    <!-- You can implement a seat map by using `amp-list`, `amp-bind` and `amp-pan-zoom` where
+    `amp-list` loads the list of seats. 
+    We are using the `transformEnd` action from [amp-pan-zoom](https://www.ampproject.org/docs/reference/components/amp-pan-zoom)
+    to update the value of transform.scale variable.
+    We display 2 types of seats: wheelchair and standard. Each seat can be available, unavailable and selected. Everytime a seat is 
+    selected we add the seat id to the `user.selectedSeats` variable.
+    -->
+     <div class="list-container">
```

Comment:
This is a massive amount of code. Can you simplify the sample to only the important bits? Maybe even git rid of the placeholder, or show the placeholder in a second example.

Reply:
I think it'd be could to simplify it to one seat category which can be selected.

Reply:
this sample is not for showing the single component but for showing how likely can be used and I think the placeholder is an important thing to show.

Reply:
That makes sense, but maybe show it in a second sample on the page.

Reply:
I think when using amp-list it's a best practice to have a placeholder, so I will keep it this way

Reply:
Sounds good. Basically try to remove everything not necessary for understanding how it works.